src/common/components/repository-row/index.js

```diff
@@ -149,15 +179,24 @@ class RepositoryRow extends Component {
   }
 
   render() {
-    const { snap, latestBuild, fullName, authStore } = this.props;
+    const {
+      snap,
+      latestBuild,
+      fullName,
+      authStore,
+      registerNameStatus
+    } = this.props;
 
     const unconfigured = true;
     const showUnconfiguredDropdown = unconfigured && this.state.unconfiguredDropdownExpanded;
-    const unregistered = true;
-    const showUnregisteredDropdown = unregistered && this.state.unregisteredDropdownExpanded;
+    const showUnregisteredDropdown = this.state.unregisteredDropdownExpanded;
     const showRegisterNameInput = (
       showUnregisteredDropdown && authStore.authenticated
     );
+    const registeredName = (
+      registerNameStatus.success ?
+      registerNameStatus.snapName : snap.store_name
+    );
 
     const isActive = showUnconfiguredDropdown; // TODO (or any other dropdown)
```

Comment:
`isActive` should be set true also when `showUnregisteredDropdown` is true, this will make the row white.